api/src/test/java/org/pmiops/workbench/api/CohortsControllerTest.java

```diff
@@ -285,7 +289,7 @@ public void setUp() {
     user = userDao.save(user);
     currentUser = user;
 
-    cdrVersion = new DbCdrVersion();
+    cdrVersion = TestMockFactory.createDefaultCdrVersion(cdrVersionDao, accessTierDao);
```

Comment:
Made a common method for CDR Version which includes Access Tier.  Otherwise I'd have to update all these tests individually.